README.md

```diff
@@ -53,18 +53,55 @@ S1APTester binary can be generated by executing below compilation steps:
 $ cd TestCntlrApp/build
 $ make cleanall
 $ make
+$ sudo cp ../lib/libtfw.so /usr/lib
+$ cd ../../
```

Comment:
In case of using S1APTester with normal magma setup, the steps of copying the libtfw.so file to /home/vagrant/s1ap-tester/bin/ is part of VM provisioning and the python scripts are importing directly from this path.

As of now, the stub binary "testCntrlr" is trying to search the libtfw.so file in default lib folder as the path is not specified with the binary. This is why we need to copy the .so file in /usr/lib/ folder.